chart/templates/grafana.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if and (.Values.grafana) (.Values.grafana.enable) -}}
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: grafana
+  namespace: argocd
+  finalizers:
+    - resources-finalizer.argocd.argoproj.io
+spec:
+  project: default
+  source:
+    repoURL: {{ .Values.repoUrl }}
```

Comment:
it might be helpful here to have some guard rails: `repoURL: {{ required "repoUrl value is required" .Values.repoUrl }}`
as per [Chart dev tips](https://helm.sh/docs/howto/charts_tips_and_tricks/)

Reply:
Makes sense, I'll add this as well!